code/test-code.sh

```diff
@@ -1,12 +1,12 @@
-#/bin/sh
+#!/bin/sh
```

Comment:
as we have the test server now we can just test it by running the scripts against that one.
otherwise we can test each sandbox too with it's import script or dump file.